coffeecard/CoffeeCard.WebApi/appsettings copy.bicep

```diff
@@ -0,0 +1,177 @@
+// {
```

Comment:
Where did this file come from?

Reply:
probably leftover file from the last time i was in this repo. removing